dhis-2/dhis-api/src/main/java/org/hisp/dhis/icon/CustomIconOperationParams.java

```diff
@@ -35,21 +35,21 @@
 import org.hisp.dhis.common.Pager;
 
 /**
- * Represents query parameters sent to {@link org.hisp.dhis.icon.IconService}.
+ * Represents query parameters sent to {@link org.hisp.dhis.icon.CustomIcon}.
  *
  * @author Zubair Asghar
  */
 @Data
 @NoArgsConstructor
-public class IconOperationParams {
+public class CustomIconOperationParams {
```

Comment:
_Operation_ is not a pattern we use and if we would use if for manipulation, not querying.
How about `IconQueryParams` ?